.gitlab-ci.yml

```diff
@@ -172,7 +173,7 @@ before_script:
     - BIN=$(readlink -f ../_install_ci/bin)/
     - LIB=$(readlink -f ../_install_ci/lib/coq)/
     - export OCAMLPATH=$(readlink -f ../_install_ci/lib/):"$OCAMLPATH"
-    - make -j "$NJOBS" BIN="$BIN" COQLIB="$LIB" COQFLAGS="${COQFLAGS}" all
+    - COQEXTRAFLAGS="${COQEXTRAFLAGS}" make -j "$NJOBS" BIN="$BIN" COQLIB="$LIB" all
```

Comment:
Revert this, the test suite only uses coq_makefile in the tests specific to it

Reply:
If we revert this, then the coq-makefile/native{1,2} tests are broken.

Reply:
But you're right that this brokes the `test-suite:base+async` CI job. I try to come with a solution.

Reply:
@SkySkimmer this should be fixed, do you agree with the fix?

Reply:
Seems ok